files/slave.dockerfile

```diff
@@ -25,13 +24,24 @@ RUN yum install -y python-pip python-devel && \
     yum groupinstall -y 'Development Tools' && \
     # Very specific version is required for older Openstack Env.
     pip install --no-cache-dir --disable-pip-version-check shade==1.12.1 && \
+
+    # Use Ansible from source until rpm available w/ fix for:
+    # https://github.com/ansible/ansible/issues/15915
+    pip install --upgrade setuptools && \
+    git clone https://github.com/ansible/ansible.git \
+        --branch=stable-2.2 --recursive --depth=0 --single-branch \
+        /usr/src/ansible && \
+    cd /usr/src/ansible && \
+    make install && \
+    
     # Keep every layer as clean (i.e. small) as possible
-    yum history -y undo $(cat /root/trans_num) && \
+    yum history -y undo $(cat /root/trans_num) ; \
     rm -f /root/trans_num && \
     yum clean all && \
     rm -rf /usr/src /usr/share/doc
 
-RUN echo "" > /etc/ansible/hosts && \
+RUN mkdir -p /etc/ansible && \
+    echo "" > /etc/ansible/hosts && \
```

Comment:
The previous rpm-based install of ansible too care of making this directory, now we gotta do it here instead.